content/docs/start/experiments.md

```diff
@@ -18,17 +18,87 @@ once they're no longer needed.
 
 ## Running experiments
 
-Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
-[compare the changes](/doc/start/metrics-parameters-plots). Let's further
-increase the number of features in the `featurize` stage to see how it compares.
+In the previous page, we learned how to tune
+[ML pipelines](/doc/start/data-pipelines) and compare the changes. In this
+section we'll use a
+[Get Started to Experiments project](https://github.com/iterative/get-started-experiments)
+to illustrate experimentation features in DVC 2.0
+
+<details>
+
+### 🆕 Click here for the instructions to install the project
+
+These commands are run in
+[`get-started-experiments`](https://github.com/iterative/get-started-experiments)
+project. You can run these commands after cloning the repository and install the
+requirements.
+
+Please clone and create a virtual environment:
+
+```console
```

Comment:
console -> dvc

start each line with $ (except output) to make highlighter work

Reply:
I thought `dvc` is specifically for DVC commands. Thank you.